lib/schema.ts

```diff
@@ -663,6 +663,81 @@ const ApiGatewayConfigSchema = z
   })
   .optional();
 
+/**
+ * Configuration for models inside the LiteLLM Config
+ * See https://litellm.vercel.app/docs/proxy/configs#all-settings for more details.
+ */
+const LiteLLMModel = z.object({
+  model_name: z.string(),
+  litellm_params: z.object({
+    model: z.string(),
+    api_base: z.string().optional(),
+    api_key: z.string().optional(),
+    aws_region_name: z.string().optional(),
+  }),
+  model_info: z
+    .object({
+      id: z.string().optional(),
+      mode: z.string().optional(),
+      input_cost_per_token: z.number().optional(),
+      output_cost_per_token: z.number().optional(),
+      max_tokens: z.number().optional(),
+      base_model: z.string().optional(),
+    })
+    .optional(),
+});
+
+/**
+ * Core LiteLLM configuration.
+ * See https://litellm.vercel.app/docs/proxy/configs#all-settings for more details about each field.
```

Comment:
I think this is fine in general but when we add some examples to the README we may want to add some additional details about the properties we expect customer may want to actually set. You've got comments on some parameters but not most and some of them seem specific to google/azure which wouldn't apply at all for LISA.

Reply:
ack. I didn't want to restrict folks' abilities to use the entire LiteLLM config- I only have as much information as the proxy docs have (which is about as accurate as I can be). I can add docs to the readme for this.